ext/adaptors/ranges-lib.tex

```diff
@@ -239,17 +244,19 @@
 
 \begin{itemdescr}
 \pnum
-\effects Equivalent to \tcode{return ranges::begin(derived()) == ranges::end(derived());}.
+\effects Equivalent to: \tcode{return ranges::begin(derived()) == ranges::end(derived());}.
 \end{itemdescr}
 
 \indexlibrary{\idxcode{operator bool}!\idxcode{view_interface}}%
 \begin{itemdecl}
-constexpr explicit operator bool() const requires ForwardRange<const D>;
+constexpr explicit operator bool() const
+  requires @\changed{ForwardRange<const D>}{requires \{ ranges::empty(derived()); \}}@;
 \end{itemdecl}
 
 \begin{itemdescr}
 \pnum
-\returns \tcode{!empty()}.
+\removed{\returns \tcode{!empty()}.}
+\added{\effects Equivalent to: \tcode{return !ranges::empty(r);}}
```

Comment:
What is `r`?

Reply:
It's what a pirate says. (cut'n'paste, fixing.)